include/item.h

```diff
@@ -101,18 +101,24 @@ u32 GetMoneyValue(struct ItemSlot* slot);
 u16 GetItemMove(u8 index);
 bool8 CanSellItem(u32 id);
 bool8 IsGummiItem(u8);
-void xxx_init_unk230_substruct(u8);
-bool8 AddGreenKecleonShopItem(u8);
-struct HeldItem *xxx_get_inv_unk230_at_809185C(u8);
-void xxx_fill_unk230_gaps();
-struct HeldItem* xxx_get_unk250_at_8091A90(u8);
-s32 xxx_count_inv_unk230();
-u32 xxx_count_non_empty_inv_unk250_8091A48();
-void ChoosePurpleKecleonShopInventory(u8);
 void sub_8090F58(void*, u8 *, struct ItemSlot *, struct unkStruct_8090F58*);
 void ShiftItemsDownFrom(s32 start);
 void ClearItemSlotAt(u32 index);
 void MoveToStorage(struct ItemSlot* slot);
+s32 CountKecleonItemShopItems(void);
```

Comment:
The in-game names are "Kecleon Shop" and "Kecleon Wares", so perhaps we can use "shop"/"wares" for function names instead of "items"/"wares". Otherwise, the existing "green" and "purple" Kecleon is also less ambiguous. Let's pick one and use that naming convention everywhere.

Reply:
Hmm good point. I'm leaning toward Kecleon Shop/Kecleon Wares.

Reply:
Works for me.